defaults/main.yml

```diff
@@ -12,9 +12,11 @@ monit_mail_enabled: false
 monit_mailserver_host: localhost
 monit_mailserver_port: 25
 
-monit_webinterface_enabled: false
+monit_webinterface_enabled: true
 monit_webinterface_bind: 0.0.0.0
```

Comment:
maybe bind to 127.0.0.1 to be sure?

Reply:
allow only for localhost.

Reply:
+ it's not opened in security groups. so it's safe.

Reply:
it's required for monit control, that's why we need it.
we can add telegraf on top of it later on